site/src/docs-components/animation/AnimationSequencesExample.scss

```diff
@@ -1,8 +1,8 @@
 @import "~@kaizen/component-library/styles/animation";
 @import "~@kaizen/design-tokens/sass/animation";
 
-$duration: calc(-1 * #{$animation-duration-deliberate}); // 700ms
-$delay: calc(-1 * #{$animation-duration-slow}); // -400ms (stagger effect)
+$duration: -#{$animation-duration-deliberate}; // -700ms
```

Comment:
This comment was incorrect, the variable is 700ms but we're using it as a negative so I've updated accordingly.